lib/matplotlib/__init__.py

```diff
@@ -1379,6 +1379,7 @@ def is_interactive():
     return rcParams['interactive']
 
 
+@cbook.deprecated("3.1")
```

Comment:
Should it get an `alternate="rcParams['tk.window_focus']"`

Reply:
sure, added